port_name: "/dev/ttyACM1"
backup_port_name: "/dev/ttyACM2"
write_msg_start_seq: "msg"
read_msg_start_seq: "sout"
use_checksum: true
verbose: true
using_reader_thread: true